buildpipeline/pipeline.json

```diff
@@ -150,7 +150,8 @@
         "TreatWarningsAsErrors": "false"
       },
       "BuildParameters": {
-        "PB_ConfigurationGroup": "Release"
+        "PB_ConfigurationGroup": "Release",
+        "PB_BuildArguments": "-stripSymbols"
```

Comment:
@dagood Can you please provide more context on this? Based upon the discussion, I assume this implies that native binaries will not have symbol information in them but instead packaged as symbols package. Is that correct? If so, how do symbol packages get generated?